homeworks/nagybalazs/5_oop/main.py

```diff
@@ -0,0 +1,56 @@
+"""
+Készíts egy Fleet osztályt, amely kezeli a Car objektumokat:
+    -   Az osztály rendelkezzen egy listával, amelyben az autók találhatóak.
+    -   Tartalmazzon metódusokat Car objektumok hozzáadására és eltávolítására a flottába/flottából.
+    -   Tartalmazzon egy metódust, amely összesíti a flotta összes autójának összes kilométerét. 
+    -   Hozz létre néhány Car objektumot, add hozzá őket a flottához, hajts végre néhány műveletet (vezetés, tankolás), 
+        jelenítsd meg az autók állapotát és a flotta összesítő adatait.
+"""
+from car_fleet_mgr import Car
+
+class Fleet:
```

Comment:
Szuper hogy szétszedted külön modulra a main-t, ahol használod a class-ok kódját, és magát a Car class definícióját.
Lehetne ez a Fleet class is egy külön fleet.py-ban, a Car pedig egy car.py-ban. és ezeket importálhatnád a main.py-ba ahol csak a main() van.
Ezzel még jobban elszeparálnád logikailag az elemeket.